src/OrchardCore.Modules/OrchardCore.Tenants/Startup.cs

```diff
@@ -50,7 +51,13 @@ public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IS
             {
                 FileProvider = new PhysicalFileProvider(contentRoot),
                 DefaultContentType = "application/octet-stream",
-                ServeUnknownFileTypes = true
+                ServeUnknownFileTypes = true,
+
+                // Cache the tenant static files for 7 days
+                OnPrepareResponse = ctx =>
+                {
```

Comment:
Maybe only if we are not in "Development".

But what happens if we revert back to "Development". Hmm, we can just clear the browser cache.